### PR TITLE
HttpStress: add docker documentation

### PR DESCRIPTION
**Pull request by eiriktsarpalis**
Per @davidsh's request adding documentation on running the stress suite in docker-compose. Minor changes in the linux stress pipeline to improve configurability.

### CODE REVIEW

File: eng/pipelines/stress/http-linux.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ pool:
 variables:
   httpStressProject: src/System.Net.Http/tests/StressTests/HttpStress
   sdkBaseImage: sdk-corefx-current
-  httpStressImage: httpstress
+  HTTPSTRESS_IMAGE: httpstress
```
Comment (davidsh) [inline eng/pipelines/stress/http-linux.yml]: I don't understand YAML sufficiently. But why is this in all capital letters but the other tags ('sdkBaseImage') not?  Why is that a change for the better?

Comment (eiriktsarpalis) [reply eng/pipelines/stress/http-linux.yml]: That's a good point. I guess it reflects the fact that the particular variable is getting consumed by docker-compose (which uses all-caps variable names). The rest just follow convention used in other pipelines in this repo. I don't have strong opinions on which is preferable.

For the sake of consistency I could maybe declare the variable using camelCase then map to the all caps name for the docker-compose task. @safern opinions?

Comment (safern) [reply eng/pipelines/stress/http-linux.yml]: I think we should try and be consistent within the yaml file, if you want to use upper case, then let's make all the variables upper case, but I think mapping them and having them as camelCase make sense.

File: eng/pipelines/stress/http-linux.yml [hunk 2]
```diff
@@ -20,11 +20,12 @@ steps:
 
 - bash: |
     cd '$(Build.SourcesDirectory)/$(HttpStressProject)'
-    docker build -t $(httpStressImage) --build-arg SDK_BASE_IMAGE=$(sdkBaseImage) --build-arg CONFIGURATION=$(BUILD_CONFIGURATION) .
+    docker build -t $(HhttpStressImage) --build-arg SDK_BASE_IMAGE=$(sdkBaseImage) --build-arg CONFIGURATION=$(BUILD_CONFIGURATION) .
```
Comment (halter73) [inline eng/pipelines/stress/http-linux.yml]: This looks like a copy/paste error.

```suggestion
    docker build -t $(httpStressImage) --build-arg SDK_BASE_IMAGE=$(sdkBaseImage) --build-arg CONFIGURATION=$(BUILD_CONFIGURATION) .
```

Comment (eiriktsarpalis) [reply eng/pipelines/stress/http-linux.yml]: Good catch, thanks!

**Pull request merged**